packages/next-plugin-treat/index.js

```diff
@@ -0,0 +1,19 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const TreatPlugin = require('treat/webpack-plugin');
+
+module.exports = (pluginOptions = {}) => (nextConfig = {}) => {
+  return Object.assign({}, nextConfig, {
+    webpack(config, { isServer }) {
+      config.plugins.push(
+        new TreatPlugin(
+          Object.assign({}, pluginOptions, {
+            outputLoaders: [MiniCssExtractPlugin.loader],
+            outputCSS: !isServer,
+          }),
+        ),
+        new MiniCssExtractPlugin(),
```

Comment:
This isnt needed - nextjs uses minicss itself, so the plugin is already configured.

https://github.com/zeit/next.js/blob/ea7c1e1dcf5b4019365e1ab3ff989801c958f0e9/packages/next/build/webpack/config/blocks/css/index.ts#L302-L317